### PR TITLE
Update docs to include latest changes

### PR DESCRIPTION
**Pull request by JAORMX**
This includes:

* `ComplianceCheckResults` additions
* Eventing support
* new display fields in objects
* scheduled scans

This also adds descriptions of the different custom resources that the operator enables as well as updating the remediations doc.

Finally, this also updates the example `ComplianceSuite` object to include a schedule and a single scan for simplicity.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,301 @@ The compliance-operator is a OpenShift Operator that allows an administrator
...
+  scanType: Node
+  profile: xccdf_org.ssgproject.content_profile_moderate
+  content: ssg-ocp4-ds.xml
+  contentImage: quay.io/complianceascode/ocp4:latest
+  rule: "xccdf_org.ssgproject.content_rule_no_netrc_files"
+  nodeSelector:
+    node-role.kubernetes.io/worker: ""
+status:
+  phase: DONE
+  result: NON-COMPLIAN
```
Comment (mrogers950) [inline README.md:104]: typo

Comment (JAORMX) [reply README.md:104]: oops!

Comment (JAORMX) [reply README.md]: Fixed. Thanks!

File: README.md [hunk 2]
```diff
@@ -4,9 +4,301 @@ The compliance-operator is a OpenShift Operator that allows an administrator
...
+In the `spec`:
+* **autoApplyRemediations**: Specifies if any remediations found from the
+  scan(s) should be applied automatically.
+* **schedule**: Defines how often should the scan(s) be run in cron format.
+* **scans** contains a list of scan specifications to run in the cluster.
+
+In the `status`:
+* **aggregatedPhase**: indicates the overall phase where the scans are at. To
+  get the results you normally have to wait for the overall phase to be `DONE`.
+* **aggregatedResult**: Is the overall verdict of the cluster.
```
Comment (mrogers950) [inline README.md:60]: The scan suite may not necessarily cover the cluster, so we probably want to say overall verdict of the suite.

File: doc/remediation-flow.md [hunk 3]
```diff
@@ -76,24 +70,15 @@ spec:
...
-  - apply: false
-    remediationName: workers-scan-no-empty-passwords
-    scanName: workers-scan
-    type: MachineConfig
-  - apply: false
-    remediationName: masters-scan-no-direct-root-logins
-    scanName: masters-scan
-    type: MachineConfig
+  aggregatedPhase: DONE
+  aggregatedResult: NON-COMPLIAN
```
Comment (mrogers950) [inline doc/remediation-flow.md:74]: typo

File: README.md (hunk 2 above)
Comment (JAORMX) [reply README.md]: Good suggestion! fixed.

File: doc/remediation-flow.md (hunk 3 above)
Comment (JAORMX) [reply doc/remediation-flow.md]: fixed

File: doc/remediation-flow.md [hunk 4]
```diff
@@ -139,7 +123,7 @@ spec:
 ### The scan-remediate-repeat flow
 The general flow is common for the cluster scan as well as the node
```
Comment (mrogers950) [inline doc/remediation-flow.md:124]: we should say Platform scans on this line too.

File: doc/remediation-flow.md [hunk 5]
```diff
@@ -139,7 +123,7 @@ spec:
...
 scan. How the remediations are represented and therefore how they are
-applied then differs for node scans and then cluster scan.
+applied then differs for node scans and then platform scan.
```
Comment (mrogers950) [inline doc/remediation-flow.md:126]: This reads a little strangely, we might say "How the remediations are represented and therefore how they are applied differs for node and platform scans."

File: doc/remediation-flow.md (hunk 4 above)
Comment (JAORMX) [reply doc/remediation-flow.md:124]: fixed

File: doc/remediation-flow.md (hunk 5 above)
Comment (JAORMX) [reply doc/remediation-flow.md]: fixed.

**Pull request merged**